CMSIS-OS/ChibiOS/I2M_OXYGEN_NF/target_windows_devices_serialcommunication_config.cpp

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include "win_dev_serial_native.h"
+
+///////////
+// UART1 //
+///////////
+
+// pin configuration for UART1
+// port: GPIOA
+// TX pin: is GPIOA_9
+// RX pin: is GPIOA_10
+// GPIO alternate pin function is 7 (see "Table 9. Alternate function mapping" in STM32F411xC and STM32F411xE datasheet)
+UART_CONFIG_PINS(1, GPIOA, 9, 10, 7)
```

Comment:
This has suffered a slight change to allow Gpio from different ports, please update or it won't work with the latest version.

Reply:
OK, done that.